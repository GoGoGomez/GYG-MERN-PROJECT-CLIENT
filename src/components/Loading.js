import React from 'react';
import loading from '../images/loading.gif'

const Loading = () => (
    <div className="Loading">
        <img src={loading} /><br />
        <h1>Loading...</h1>
    </div>
)

export default Loading;