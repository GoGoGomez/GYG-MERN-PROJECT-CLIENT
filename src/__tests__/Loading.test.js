import React from "react";
import renderer from "react-test-renderer";
import Loading from "../components/Loading";

test("Loading component renders correctly", () => {
  const component = renderer.create(<Loading />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
