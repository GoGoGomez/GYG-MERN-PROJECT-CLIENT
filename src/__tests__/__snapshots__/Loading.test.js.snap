// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component renders correctly 1`] = `
<div
  className="Loading"
>
  <img
    src="loading.gif"
  />
  <br />
  <h1>
    Loading...
  </h1>
</div>
`;
